easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -34,15 +42,28 @@ osdependencies = [
     ('pam-devel', 'libpam0g-dev')
 ]
 
+start_dir = "%(builddir)s/rstudio-%(version)s"
+separate_build_dir = False
 build_type = "Release"
-local_dep_dir = "%(builddir)s/rstudio-%(version)s/dependencies/common"
-preconfigopts = ("(cd %s && ./install-dictionaries && "
-                 "./install-pandoc && ./install-mathjax) && ") % local_dep_dir
+
+preconfigopts = " && ".join([
+    "cd %(builddir)s/rstudio-%(version)s/dependencies/common",
+    "./install-cef",
+    "./install-dictionaries",
+    "./install-mathjax",
+    "./install-pandoc",
+    "./install-packages",
+    "cd ../..",
+    "",  # To append &&
+])
+
 configopts = "-DRSTUDIO_TARGET=Server -DRSTUDIO_BOOST_SIGNALS_VERSION=2"
 
 sanity_check_paths = {
-    'files': ["bin/rstudio-server"],
+    'files': ["bin/rstudio-server", "bin/rsession", "bin/rserver", "bin/rserver-pam", "bin/rpostback"],
     'dirs': ['bin', 'extras', 'resources', 'www', 'www-symbolmaps', 'R'],
 }
 
+sanity_check_commands = ["rstudio-server verify-installation"]
```

Comment:
You could add the following checks as well :
```
sanity_check_commands = [
    "rsession --verify-installation=1",
    "rserver --verify-installation=1",
    "rstudio-server verify-installation",
]
```

Reply:
Thanks for the tip, I added these in https://github.com/easybuilders/easybuild-easyconfigs/pull/12544/commits/a335b99d2dec1fbbf050cb24d2fd3300399640b7